starter_code/css/mobile.css

```diff
@@ -0,0 +1,217 @@
+@media(min-width:320px) {
+ /*small portrait stylez*/
+ 
+body {
+  font-size: 14px;
+  line-height: 1;
+  font-weight: 300;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+.container {
+  width: 100%;
+ }
+
+.logo {
+  font-size: 36px;
+  font-weight: 800;
+ }
+
+header {
+  padding: 3px 0;
+  height: 15vh;
+  box-sizing: border-box;
+ }
+
+ header .logo {
+  float: none;
+  text-align: center;
+  padding-bottom: 5%;
+}
+
+
+header nav {
+	float: none;
```

Comment:
Had you just moved the `float: right` to the tablet/desktop style sheet, you wouldn't need this rule here.